pkg/datagatherer/k8s/dynamic_test.go

```diff
@@ -67,18 +68,20 @@ func asUnstructuredList(items ...*unstructured.Unstructured) *unstructured.Unstr
 }
 
 func TestNewDataGathererWithClient(t *testing.T) {
+	ctx := context.TODO()
```

Comment:
I'm not sure why this is TODO rather than just an empty context? (`context.Context`)

My understanding was that TODO was for when the a higher context was to be supplied from higher up at a later date.

Reply:
you are right! I have changed to [context.Background()](https://golang.org/pkg/context/#Background).